lib/evented/index.js

```diff
@@ -19,7 +19,37 @@ var ArrayProto = Array.prototype;
 var nativeIndexOf = ArrayProto.indexOf;
 var slice = ArrayProto.slice;
 
-function bind(event, fn) {
+/**
+ * Initialize a custom error handler or default to the BV Loader error reporting
+ * regardless of application. This will automatically be called at the first
+ * invocation of the ".on" method, and can be 
+ * event
+ * @param {Function} fn - the custom callback to be invoked whenever an error is
+ *  detected as the result of a trigger event
+ */
+function setErrorHandler(fn) {
+  this.errorHandler = function(e) {
+    global.console.error(e);
+
+    if (fn) {
+      fn.apply(this, slice.call(arguments))
+    } else {
+      if (global && global.BV && global.BV._private) {
```

Comment:
Sending errors automatically to a global BV instance is a poor integration, and creates a problem if someone other than BV uses this library on a product page where BV also resides.